Makefile

```diff
@@ -33,7 +33,13 @@ check: ## Lint code
 build: ## Build binary
 	@echo -e "\033[32mBuilding package...\033[0m"
 	mkdir -p bin
-	$(DOCKER_CMD) go build -v -o bin/machine-api-operator cmd/main.go
+	$(DOCKER_CMD) go build -v -o bin/machine-api-operator github.com/openshift/machine-api-operator/cmd
```

Comment:
this won't work outside of CI.

Reply:
Since you need to set the GOPATH?

Reply:
Yes. the whole point of using docker for building is that you don't need to set anything on host.

Reply:
Undone